<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Товар</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="2" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Оборудование</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tw_machine">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Производитель</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Принять</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>На складе</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QLabel" name="lb_photo">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>Фото</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Производитель</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_name"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Артикул</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_article">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Цена продажи</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="le_price">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="le_manufacturer">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Фото</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_6">
          <property name="text">
           <string>С сайта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_5">
          <property name="text">
           <string>Изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_note"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Заметка</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_vaendor_name"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Название пост.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pb_copy"/>
  </widget>
  <action name="pb_copy">
   <property name="text">
    <string>Скопировать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_manufacturer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>-60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_change_photo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>-50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_del_photo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>-78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_machine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>-51</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_del_machine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>-42</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_acc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_can()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_web_photo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_copy</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_view_manufacturer()</slot>
  <slot>ui_change_photo()</slot>
  <slot>ui_del_photo()</slot>
  <slot>ui_view_machine()</slot>
  <slot>ui_del_machine()</slot>
  <slot>ui_acc()</slot>
  <slot>ui_can()</slot>
  <slot>ui_web_photo()</slot>
  <slot>ui_copy()</slot>
 </slots>
</ui>
